src/styles/sidebar/components/AnnotationPublishControl.scss

```diff
@@ -52,6 +54,7 @@
   // Safari. See https://github.com/hypothesis/client/issues/2302
   &__menu-wrapper {
     @include layout.row;
+    @include u.transition--fast('background-color');
```

Comment:
I think this may be a long standing bug... the arrow next to the publish button does not animate on hover. It's actually unrelated to this change, but I noticed it doing these changes. I can create follow PR for it.

Reply:
Why is `'background-color'` quoted here but in another usage it is like so

`@include u.transition--fast(color, background-color, box-shadow);`

Reply:
a mistake, it should just be text w/o quote (i think)